Kickstarter-iOS/Features/Dashboard/Controller/DashboardViewController.swift

```diff
@@ -61,6 +64,8 @@ internal final class DashboardViewController: UITableViewController {
     super.viewWillDisappear(animated)
 
     self.viewModel.inputs.viewWillDisappear()
+
+    self.removeDeprecationWarning()
```

Comment:
I used your idea of view's appearing and disappearing which is the correct approach. Good idea to keep these changes within the `DashboardViewController`. I tried it in `RootTabBarViewController` and it would be messy turning it off there for each tab.